src/canreg/client/CanRegClientApp.java

```diff
@@ -139,6 +139,20 @@ public void changePassword(String encrypted) throws SecurityException, RemoteExc
         }
     }
 
+    /**
+     * Check if the hashed password correspond to the password in the database
+     * 
+     * @param username username
+     * @param password password of the user
```

Comment:
Would it be more accurate to rename the parameter 'password' into 'encryptedPassword' ?